packages/actor-init-sparql/lib/ActorInitSparql.ts

```diff
@@ -69,6 +71,28 @@ export class ActorInitSparql extends ActorInitSparqlBrowser {
     return existsSync(`${__dirname}/../test`);
   }
 
+  // Function to convert a URL like 'hypermedia@http://user:passwd@example.com to an IDataSource
```

Comment:
Could you transform this comment into actual jsdoc? So `/** ...`, with `@param` and stuff.